### PR TITLE
Mepts fix import pvls cohort queries

### PR DESCRIPTION
**Pull request by cfaife**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxPvlsCohortQueries.java [hunk 1]
```diff
@@ -11,6 +11,8 @@
  */
 package org.openmrs.module.eptsreports.reporting.library.cohorts;
 
+import static org.openmrs.module.eptsreports.reporting.utils.EptsReportConstants.*;
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxPvlsCohortQueries.java:14]: Can this be done individual imports as oppossed to use of wild cards?

Comment (psmatsinhe) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxPvlsCohortQueries.java:14]: We did this way cause the module was failing to initialize in the server using the individual import

**Comment by psmatsinhe** [comment]
> _No description provided._

@cfaife Let's close this one as we did not see that problem again... I believe it was those crazy that I cant tell...

**Comment by psmatsinhe** [comment]
> _No description provided._

@cfaife Let's close this one as we did not see that problem again... I believe it was those crazy that I cant tell...

**Pull request closed without merging**